green_cli/common.py

```diff
@@ -535,6 +535,16 @@ def bumpfee(session, previous_txid, fee_multiplier, subaccount, wait, timeout):
     details['fee_rate'] = int(previous_transaction['fee_rate'] * fee_multiplier)
     return _send_transaction(session, details, wait, timeout)
 
+@green.command()
+@click.argument('address', type=str, expose_value=False, callback=details_json)
+@click.argument('message', type=str, expose_value=False, callback=details_json)
+@with_login
+@print_result
+def signmessage(session, details):
+    """Sign a message"""
+    auth_handler = gdk.sign_message(session.session_obj, json.dumps(details))
```

Comment:
Is there a reason we didn't use `@with_gdk_resolve`?

Reply:
no, changed to use it